website/docs/commands/api-compatibility.md

```diff
@@ -387,7 +387,7 @@ Note that this list is subject to change as we continue to expand our API comman
 |  | GETSET | ➖ |  |
 |  | [INCR](raw-string.md#incr) | ➕ |  |
 |  | [INCRBY](raw-string.md#incrby) | ➕ |  |
-|  | INCRBYFLOAT | ➖ |  |
+|  | [INCRBYFLOAT](raw-string.md#incrbyfloat) | ➖ |  |
```

Comment:
@Vijay-Nirmal Presumably "-" meant to be changed to "+".

Currently `INCRBYFLOAT` may look like not implemented with the minus sign
https://microsoft.github.io/garnet/docs/commands/api-compatibility

Reply:
This PR should fix this, if accepted https://github.com/microsoft/garnet/pull/757